internal/civisibility/utils/codeowners_test.go

```diff
@@ -0,0 +1,166 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024 Datadog, Inc.
+
+package utils
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCodeOwners(t *testing.T) {
+	codeOwners := GetCodeOwners()
+	assert.NotNil(t, codeOwners)
+	assert.Equal(t, codeOwners.Match("/appsec").GetOwnersString(), "[\"@DataDog/asm-go\"]")
```

Comment:
> Anyway, could we use a fixture instead of the real CODEOWNERS files? I can see in the future this test failing because we decided to change a team's name or a path's ownership.

Yes, I agree with this not being ideal as it depends on an actual used file. I've removed that test as the only unique thing it was testing was the actual fetching of the repo's CODEOWNERS file (and that made it inherently "flaky" as unrelated changes to that file could make it fail in the future).